lib/node_modules/@stdlib/blas/base/ssyr/src/ssyr.c

```diff
@@ -0,0 +1,80 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/base/ssyr.h"
+#include "stdlib/blas/base/shared.h"
+
+/**
+* Performs the symmetric rank 1 operation `A = α*x*x^T + A`.
+*
+* @param order    storage layout
+* @param uplo     specifies whether the upper or lower triangular part of the symmetric matrix `A` should be referenced
+* @param N        number of elements along each dimension of `A`
+* @param alpha    scalar
+* @param x        input vector
+* @param strideX  `x` stride length
+* @param A        input matrix
+* @param LDA      stride of the first dimension of `A` (a.k.a., leading dimension of the matrix `A`)
+*/
+void API_SUFFIX(c_ssyr)( const CBLAS_LAYOUT order, const CBLAS_UPLO uplo, const CBLAS_INT N, const float alpha, const float *X, const CBLAS_INT strideX, float *A, const CBLAS_INT LDA ) {
+	CBLAS_INT ix0;
+	CBLAS_INT ix1;
+	CBLAS_INT i0;
+	CBLAS_INT i1;
+	CBLAS_INT ox;
+	float tmp;
+
+	if ( N == 0 || alpha == 0.0f ) {
+		return;
+	}
+	ox = STDLIB_BLAS_BASE_STRIDE2OFFSET( N, strideX );
+	if (
+		( order == CblasRowMajor && uplo == CblasLower ) ||
+		( order == CblasColMajor && uplo == CblasUpper )
+	) {
+		ix1 = ox;
+		for ( i1 = 0; i1 < N; i1++ ) {
+			if ( X[ ix1 ] != 0.0f ) {
+				tmp = alpha * X[ ix1 ];
+				// oa = (sa1*i1);
```

Comment:
@aman-095 Should this commented line be here?

Reply:
No, I forgot to remove it. Done.